packages/backend-app-api/src/services/implementations/httpRouter/createCredentialsBarrier.ts

```diff
@@ -59,13 +65,33 @@ export function createCredentialsBarrier(options: {
   const unauthenticatedPredicates = new Array<(path: string) => boolean>();
   const cookiePredicates = new Array<(path: string) => boolean>();
 
-  const middleware: RequestHandler = (req, _, next) => {
+  // Default rate limit is 100 requests per 15 minutes
```

Comment:
Tbh I think this is quite a bit more aggressive than we need, something like 60/min or so is the kinda number I had in mind. Really just aiming to block trivial DoS attacks and similar

Reply:
Updated 🙏🏻